community/server-api/src/main/java/org/neo4j/server/rest/repr/Serializer.java

```diff
@@ -76,6 +76,11 @@ final void serialize( ListWriter list, ListRepresentation value )
         list.done();
     }
 
+    final String relativeBoltUri(String boltPort){
+        String bolt = baseUri.toString().replaceFirst( baseUri.getScheme(), "bolt" );
```

Comment:
This seems a bit backward to me. Why throw away the parsed URI and use string manipulation (replaceFirst, length etc.) when when you can neatly extract the host part out of the URI with a method call?